doc/source/developing/creating_derived_fields.rst

```diff
@@ -219,6 +219,29 @@ In this case, we already know what the ``center`` of the sphere is, so we do
 not set it. Also, note that ``center`` and ``bulk_velocity`` need to be
 :class:`~yt.units.yt_array.YTArray` objects with units.
 
+If you are writing a derived field that uses a field parameter that changes the
+behavior of the field depending on the value of the field parameter, you can
+make yt test to make sure the field handles all possible values for the field
+parameter using the ``ValidateParameter`` field validator.
```

Comment:
The example above introduces `ValidateParameter`, but if I understand correctly, what's different here is that you have provided a dictionary of possibly values to test. I suggest explicitly mentioning that that's what's being added here. Additionally, it might be helpful to change the above example to `ValidateParameter(['center', 'bulk_velocity'])`, since it is not clear why you need to specify the values as a dict instead of just a list in this example.

Reply:
To be clear, I mean the example above the one you edited.